src/app/containers/Include/index.jsx

```diff
@@ -14,6 +25,64 @@ const IncludeContainer = ({ html, type }) => {
 
   const shouldNotRenderInclude = !enabled || !html || !supportedTypes[type];
 
+  const createAppendScriptTag = (code, src) => {
+    return new Promise((resolve, reject) => {
+      const script = document.createElement('script');
+      if (src) {
+        script.src = src;
+        // eslint-disable-next-line func-names
+        script.onload = function () {
+          resolve();
+        };
+      } else if (code) {
+        script.appendChild(document.createTextNode(code));
+      }
+      // eslint-disable-next-line func-names
+      script.onerror = function () {
+        reject();
+      };
+      document.body.append(script);
+      if (code) {
+        resolve();
+      }
```

Comment:
It might also be beneficial to pull this out into a helper function. Something that can be easily unit tested in isolation with basic inputs and expected outputs. This would also clean up the container a bit more.